src/content/documentation/publish/version-compatibility.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: sidebar
+title: Firefox version compatibility
+permalink: /documentation/publish/version-compatibility/
+topic: Publish
+tags: [add-ons, intermediate, tutorial, webextensions, amo, distribution]
+contributors: [dotproto]
+last_updated_by: dotproto
+date: 2023-10-05
+---
+
+<!-- Page Hero Banner -->
+
+{% capture page_hero_banner_content %}
+
+# Firefox version compatibility
+
+Learn how to customize Firefox version compatibility settings on addons.mozilla.org and when to use this feature.
+
+{% endcapture %}
+{% include modules/page-hero.liquid,
+    content: page_hero_banner_content
+%}
+
+<!-- Content with Table of Contents Module -->
+
+{% capture content_with_toc %}
+
+## AMO settings vs. `browser_specific_settings`
+
+## Manifest `browser_specific_settings` key
+
+## AMO compatibility setting
+
+You can control the versions of Firefox that can install your add-on by configuring a version's compatibility settings. 
+
+1. Visit the [Add-on Developer Hub](https://addons.mozilla.org/en-US/developers/). <br/><br/> <img src="/assets/img/publish/configure_versions_dev_hub.png" alt="Screen showing the My Add-ons list in the Add-on Developer Hub" style="box-shadow:0 0 0.5em gray;" />
```

Comment:
@rebloor, I seem to recall that in another review you mentioned that you preferred the use of Markdown-style image includes over HTML ones. Is that correct?

The main reason I was leaning towards using HTML was because it allows us to add a box shadow or border to the image, which helps the reader differentiate between screenshot content and article text.

Reply:
The usual best practice is to include the border in the imagerather than switching to HTML. This was how images were presented priginally, I think I'm bummed into a couple of legacy examples recently. However, given that we've gone in this direction and it now encompasses many of the images, I suggest we continue.

Reply:
That seems reasonable to me. Do you have a preference for how we should handle screenshots on other pages? I'm open to either baking the border into the image or using CSS to decorate images before upload. If we decide to embrace CSS, I'd like to create a [shortcode](https://www.11ty.dev/docs/shortcodes/) or convention in Markdown to make our image handling more consistent.